recipes/hklpy2/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "hklpy2" %}
+{% set version = "0.0.23" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hklpy2-{{ version }}.tar.gz
+  sha256: 520fadf2a358feead3c72fd9fc675039159a3b6cfc79e2475604faec5640bb6a
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  entry_points:
+    - no_op = hklpy2.backends.no_op:NoOpSolver
+    - hkl_soleil = hklpy2.backends.hkl_soleil:HklSolver
+    - th_tth = hklpy2.backends.th_tth_q:ThTthSolver
+  noarch: python
+
+requirements:
+  host:
+    - "python >=3.10"
+    - setuptools-scm >=8
+    - setuptools >=64.0
+    - pip
+  run:
+    - "python >=3.10"
```

Comment:
```suggestion
    - python >=3.10
    - setuptools-scm >=8
    - setuptools >=64.0
    - pip
  run:
    - python >=3.10
```

Reply:
In YAML, 3.10 == 3.1 ; it's a float

Reply:
They are not necessary here in the recipes.

Reply:
Done